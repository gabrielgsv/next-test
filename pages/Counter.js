import React from "react";

export default props => {
  return (
    <div>
      <h2>VocÃª tem {props.count} counters.</h2>
    </div>
  );
};
