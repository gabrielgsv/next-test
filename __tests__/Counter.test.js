import React from "react";
import { shallow } from "enzyme";

import Counter from "../pages/Counter";

describe("Testing Counter Component", () => {
  it("shold render correctly", () => {
    const wrapper = shallow(<Counter count={5} />);
    expect(wrapper).toMatchSnapshot();
    wrapper.setProps({ count: 10 });
    expect(wrapper).toMatchSnapshot();
  });
});
