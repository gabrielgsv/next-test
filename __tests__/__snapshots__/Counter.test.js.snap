// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Counter Component shold render correctly 1`] = `
<div>
  <h2>
    Você tem 
    5
     counters.
  </h2>
</div>
`;

exports[`Testing Counter Component shold render correctly 2`] = `
<div>
  <h2>
    Você tem 
    10
     counters.
  </h2>
</div>
`;
